### PR TITLE
document option to prevent unregistered user spend

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Prevent unregistered users from spending credits
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A how to guide to configuring your org so that unregistered users will not be able to perform actions that spend credits3
```
Comment (BrennanMacKay) [inline jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc:9]: Is the `3` on `credits3` a typo or part of the markup?

File: jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+:icons: font
+:experimental:
+
+It is possible for _unregistered_ users to trigger pipelines, and therefore, spend credits. Use the **Prevent unregistered user spend** option to only allow registered users the ability to trigger work that consumes credits. For a full breakdown of the various ways pipelines can be triggered, see the xref:triggers-overview[Triggers overview].
+
+[NOTE]
+====
+An _unregistered_ user is how CircleCI tracks users who have triggered builds but have not signed up for a CircleCI account. Unregistered users **do** count as link:https://support.circleci.com/hc/en-us/articles/360034783154[active users].
```
Comment (BrennanMacKay) [inline jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc:17]: It's described in the next linked document (https://support.circleci.com/hc/en-us/articles/360037744473-What-is-an-Unregistered-User-) but Unregistered users will map to a single "Unregistered User", though I'm not sure if we need to make that more clear here.

**Review by BrennanMacKay — APPROVED** [review]
A couple minor comments but this looks accurate

**Review by rcircleci — APPROVED** [review]

File: jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc:9]: typo! thanks

File: jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc (hunk 2 above)
Comment (rosieyohannan) [reply jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc:17]: @BrennanMacKay so multiple unregistered for an org would just be counted as a single active user?

Comment (BrennanMacKay) [reply jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc:17]: Yep that's correct, I'm not sure exactly where that mapping occurs but we have a single unregistered user actor id that all of the unregistered users are mapped to. This is what shows up as the `Unregistered User` in the user list.

Comment (rosieyohannan) [inline jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc:17]:
```suggestion
An _unregistered_ user is how CircleCI tracks users who have triggered builds but have not signed up for a CircleCI account. Unregistered user activity within an org is mapped to, and counts as a single link:https://support.circleci.com/hc/en-us/articles/360034783154[active user].
```

**Pull request merged**